docs/en/observability/create-alerts.asciidoc

```diff
@@ -3,14 +3,18 @@
 
 [IMPORTANT]
 ====
-Make sure to set up alerting in {kib}. For details, see
+Ensure Alerting is already set up in {kib}. For details, see
 {kibana-ref}/alerting-setup.html[Setup and prerequisites].
 ====
 
-Within the Logs, Metrics, and Uptime apps, alerting enables you to detect
-complex *conditions* defined by a *rule*. When a condition is met, the rule
+Alerting enables you to detect complex *conditions* defined by a rule
+within the Logs, Metrics, and Uptime apps. When a condition is met, the rule
 tracks it as an *alert* and responds by triggering one or more *actions*.
 
+Listed on the <<view-observability-alerts,Alerts>> page are all of your
+observability alerts that have met a condition defined by a rule you created
+using the Logs, Metrics, Uptime, or APM apps.
+
```

Comment:
I can not comment on the lines below. I refer to this part. I don't see apm alerts in the list. 

```
* <<logs-threshold-alert,Logs threshold rule>>
* <<infrastructure-threshold-alert,Infrastructure threshold rule>>
* <<metrics-threshold-alert,Metrics threshold rule>>
* <<monitor-status-alert,Monitor status rule>>
* <<tls-certificate-alert,TLS certificate rule>>
* <<duration-anomaly-alert,Uptime duration anomaly rule>>
```

Reply:
At the moment, the APM alerts are documented separately in the [Kibana](https://www.elastic.co/guide/en/kibana/current/apm-alerts.html) docs.